toolchain/sem_ir/formatter.cpp

```diff
@@ -710,6 +716,12 @@ class Formatter {
     }
   }
 
+  // Print ClassDecl with type-like semantics even though it lacks a type_id.
+  auto FormatInstructionLHS(InstId inst_id, ClassDecl /*inst*/) -> void {
+    FormatInstName(inst_id);
+    out_ << ": type = ";
```

Comment:
```suggestion
    out_ << " = ";
```

Let's leave off the type because this instruction doesn't have one. (This is what we used to do for instructions that had values but not types, back before that functionality was removed.)